modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -119,6 +119,45 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-172891. Verify that the start date cannot be equals to the current date"
+	@priority = "5"
+	test StartDateCannotBeEqualsToCurrentDate {
+		task ("Given I go to add a new MDF Request and fill all Goals form with my information") {
+			PRMMarketing.gotoAddNewMDF();
+
+			PRMMDFRequest.fillGoalsPage(
+				accountName = "Company Name 1",
+				additionalOptions = "6.x to DXP Migration/Upgrade,Migration from competitor platform",
+				campaignDescription = "Campaign Description",
+				campaignName = "Campaign Name",
+				countryName = "Global",
+				goalsOptions = "Lead generation",
+				targetMarketOptions = "Education,Energy",
+				targetRolesOptions = "Administrator");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("When a new activity is added") {
+			PRMMDFRequest.addNewActivity();
+		}
+
+		task ("And a date is set on 'Start Date' field") {
+			PRMMDFRequest.insertDate(
+				dateFormat = "MMddYYYY",
+				dateRange = "-1",
+				fieldName = "startDate");
+		}
+
+		Click(
+			key_fieldName = "activities[0].endDate",
+			locator1 = "PRMMDFRequest#TEXT_FIELD");
+
+		task ("Then the start date need to be diferent of acutal date") {
```

Comment:
Hi @mateussandes23, could you please rename this task? I think that it can be "Then an error message will be displayed". You don't need to specify the error on task description, since you are specifying the error on the macro below.